bin/problem.py

```diff
@@ -438,6 +441,23 @@ def build_program(p):
 
         problem._validators[key] = validators
         return validators
+    
+    # TODO Q from Thore: ok to use self here instead of problem?
+    def get_expectations_registry(self):
```

Comment:
It's probably best to stay consistent with existing code, so I would use `problem` here instead of `self`, unless we rename the first parameter of all class methods from `problem` to `self`. I don't mind either way, as long as it's consistent, so this decision is better for Ragnar to make :slightly_smiling_face:

Reply:
I'm not really a fan of `self` somehow. I think it's because it's not always clear in which class you're currently reading code with the big functions everywhere, so naming things after what they are seems clearer generally. (Especially given that `self` doesn't really mean anything special inside the member functions anyway.)

So yes, I'd prefer `problem`.